app/models/basket.rb

```diff
@@ -0,0 +1,5 @@
+class Basket < ApplicationRecord
+  belongs_to :user
+  has_many :subscriptions
+  has_many :basket_items
```

Comment:
needs dependent destroy to delete basket items

Reply:
Agreed, will add dependency